programs/survival/vines_instant_mine.sc

```diff
@@ -0,0 +1,29 @@
+///
+// Vines Instant Mine
+// by BisUmTo
+// (Carpet Mod 1.4.9)
+//
+// Shears will instant-mine vines
+///
+
+__config() -> {'stay_loaded' -> true};
+
+_vines_instant_mine(player, block) -> (
+    g = player ~ 'gamemode';
+    holds = player ~ 'holds';
+    if(g == 'spectator' || g == 'creative' || !holds || block != 'vine', return());
+    [item, count, nbt] = holds;
+    if(item!='shears',return());
+    if(g == 'adventure' && !nbt:'CanDestroy' ~ '"minecraft:vine"', return());
+    schedule(0,
```

Comment:
Correct me if this is wrong, but shouldn't it be scheduled for 1 tick delay? cos it takes 1 tick for a block to be instamined, but im not sure if this already does that.

Reply:
Im talking about the bottom line.

Reply:
The current code breaks the block at the end of the current gametick, so you can mine another block in the next tick (20 blocks mined each second).
I think that's correct, but I'm not 100% sure

Reply:
ok

Reply:
I can't close this.